.gitignore

```diff
@@ -92,3 +92,21 @@ contiki-cc2530dk.lib
 # rl78 build artifacts
 *.eval-adf7xxxmb4z
 *.eval-adf7xxxmb4z.srec
+
+# cscope files
+cscope.*
+
+# vim swap files
```

Comment:
Sure, I'll remove the gitignore patch from this PR.

I'll send it in a separated PR since it might be useful for others.